sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -534,7 +534,7 @@ async Task GetScopeProperties (int msg_id, int scope_id, CancellationToken token
 				});
 				SendResponse (msg_id, Result.Ok (o), token);
 			}
-			catch (Exception exc) {
+			catch {
```

Comment:
Aren't these a semantic change, to filter on the type or not, and this should be `catch (Exception)`?
I'll have to look it up..

Reply:
Yes seems to be a semantic. Perhaps good one though.